gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/ExploreService.java

```diff
@@ -150,4 +151,8 @@ private boolean hasTimeAggregations(ExploreRequest request) {
   private boolean hasTimeAggregationsAndGroupBy(ExploreRequest request) {
     return !request.getTimeAggregationList().isEmpty() && !request.getGroupByList().isEmpty();
   }
+
+  private boolean timeRangeSet(ExploreRequest request) {
+    return request.getStartTimeMillis() != 0L && request.getEndTimeMillis() != 0L;
```

Comment:
Can we instead make the time range field optional in GRPC? Introducing optional shouldn't be a breaking change

Reply:
All fields are optional in proto3. Do you mean introducing oneOf instead https://stackoverflow.com/questions/42622015/how-to-define-an-optional-field-in-protobuf-3?

Reply:
Done